app/src/main/java/com/toast/android/gamebase/sample/gamebase_manager/GamebaseAuth.kt

```diff
@@ -157,6 +142,14 @@ private fun handleLoginSuccess(
 ) {
     printLoginSuccess(TAG, authToken)
 
+    // TODO: [Fix me] The user level information can be set after login to the game has been made.
+    // https://docs.nhncloud.com/en/Game/Gamebase/en/aos-etc/#analytics
+    val dummyUserData = GameUserData(99).apply {
```

Comment:
[MAY] 변수명이 이미 dummyUserData라서 큰 문제는 없겠지만, '99'라는 레벨 값 역시 더미라는 것을 조금 더 확실히 하기 위해 변수에 한번 할당하는 것은 어떨까 합니다.
또한 로그인 후 호출해야 하는 초기 레벨값은 해당 유저의 현재 레벨임을 알 수 있도록 예시로는 길지만, 서버에서 받아온 레벨이라는 이름의 변수를 제안 해봤습니다.

```suggestion
    val sampleInitLevelFetchedByGameServer = 99
    val dummyUserData = GameUserData(sampleInitLevelFetchedByGameServer).apply {
```